### DISCUSSION TITLE
Missing notification

### DISCUSSION THREAD

**Discussion by see7e** [community/community#70659] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The notifications are acting like there is an unread (participating) news and I can't figure out how to remove the missing element. This showed up quite some time ago and didn't vanished.

![image](https://github.com/community/community/assets/15942792/3aada194-b131-43d0-b156-b8bf68956c27)

![image](https://github.com/community/community/assets/15942792/2df6cb2a-bd95-4507-83f4-650507d3dcd1)

**Comment by Mertozturkk** [comment]
I have the same situation and it never gets better. This situation is starting to get on my nerves.
